src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: 100%;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-size: cover;
+  height: 12px;
+  width: 12px;
+  margin: 2px 6px 2px 2px;
+}
+
+.stars--5 .stars__star,
+.stars--4 .stars__star:nth-child(1),
+.stars--4 .stars__star:nth-child(2),
+.stars--4 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(4),
+.stars--3 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(3),
+.stars--2 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(2),
+.stars--1 .stars__star:nth-child(1) {
```

Comment:
you can use a formula like :nth-child(-n + #stars-number) to acheive the same in a cleaner way

Reply:
yikes tests 🤢